src/integration-test/javascript/e2e/homepageSpec.js

```diff
@@ -0,0 +1,7 @@
+describe('Demo Protractor Test', function () {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["describe" is not defined.](https://www.codacy.com/app/martzrp/booktrackr/file/2628357651/issues/source?bid=3283479&fileBranchId=3317266#l1)